scripts/run_docker_build.sh

```diff
@@ -61,5 +61,5 @@ find conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt"
     | xargs -n1 cat | grep -v -e "^#" -e "^$" | \
     xargs -r yum install -y
 
-conda-build-all /conda-recipes --matrix-conditions "numpy >=1.10" "python >=2.7,<3|>=3.4"
+conda-build-all /conda-recipes --matrix-conditions "numpy >=1.10" "python >=2.7,<3|>=3.4" "r-base ==3.3.2"
```

Comment:
Does that mean I just need to change this constraint to r instead of r-base? Will this be persisted to the feedstocks?